doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -150,6 +150,10 @@ bash configure --with-boot-jdk=/home/jenkins/bootjdks/jdk11
 ```
 :warning: The path in the example --with-boot-jdk= option is appropriate for the Docker installation. If not using the Docker environment, set the path appropriate for your setup, such as "<my_home_dir>/bootjdk11" as setup in the previous instructions.
 
+:pencil: Configuring and building is not specific to OpenJ9 but uses the OpenJDK build infrastructure with OpenJ9 added in.
+Many other configuration options are available, including options to increase the verbosity of the build output to include command lines (`LOG=cmdlines`) or other info or debug information.
+For more information see the OpenJDK build instructions https://htmlpreview.github.io/?https://raw.githubusercontent.com/openjdk/jdk/master/doc/building.html#troubleshooting .
```

Comment:
This should link to the jdk11 documentation, not jdk.
Suggest that "OpenJDK build instructions" name the link:
```
For more information see the [OpenJDK build instructions](https://htmlpreview.github.io/?https://raw.githubusercontent.com/openjdk/jdk/master/doc/building.html#troubleshooting).
```

Reply:
Updated.